protocol/v2/qbft/instance/round_change.go

```diff
@@ -120,6 +124,9 @@ func (i *Instance) uponChangeRoundPartialQuorum(ctx context.Context, logger *zap
 	if err != nil {
 		return errors.Wrap(err, "failed to hash instance start value")
 	}
+
+	i.metrics.RecordRoundChange(ctx, newRound, partialQuorumReason)
```

Comment:
What do you think about moving this to the `uponRoundChange` method for consistency? Now, when I read it, from first sight, I feel like it's recorded only for `justifiedReason` until I check the contents of `uponChangeRoundPartialQuorum`